src/inmanta/module.py

```diff
@@ -1537,10 +1537,12 @@ def __str__(self) -> str:
 class ProjectPipConfig(BaseModel):
     """
     :param use_config_file: Indicates whether the pip configuration files have to be taken into account when installing
-                            Python packages.
+        Python packages.
+    :param index_url: List of pip indexes to use project-wide.
     """
 
     use_config_file: bool = False
+    index_url: List[str] = []
```

Comment:
Why do we use the singular word here, while it actually has the type list?
```suggestion
    index_urls: List[str] = []
```

Reply:
I don't have a strong opinion about this, I went with the naming suggested in the ticket, and it actually is consistent with the naming of other options with type List in https://docs.inmanta.com/inmanta-service-orchestrator/latest/reference/projectyml.html#project-yml (modulepath, repo, relation_precedence_policy)

Reply:
Yes. I see. But I still think it would be more clear to make it plural. I would not care about the consistency issue.

* `modulepath` and `repo` will be removed when we remove support for V1 modules. 
* `relation_precedence_policy` should be singular I think. One policy can consist of multiple rules.

So if we get this one right, we have an easy migration path to a config file that is consistent.

@sanderr @wouterdb Any opinions on this?